mne/realtime/mock_lsl_stream.py

```diff
@@ -8,7 +8,7 @@
 from ..io import constants
 
 
-class MockLSLStream:
+class MockLSLStream(object):
```

Comment:
I know that with python, we no longer need to include object here, is this done to be explicit?

Reply:
Yeah and consistent with the rest of the codebase. Maybe someday we will remove them all